src/components/editProfileModel/editProfileModel.js

```diff
@@ -0,0 +1,91 @@
+import { useState } from "react";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+import Modal from "@mui/material/Modal";
+import { TextField } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import IconButton from "@mui/material/IconButton";
+import PhotoCamera from "@mui/icons-material/PhotoCamera";
```

Comment:
Using barrel imports for `mui` will be helpful, there'll be a lo of them in almost every comp

Reply:
i'll try to change it in next pr